client/post-editor/editor-ground-control/index.jsx

```diff
@@ -55,20 +60,71 @@ export default React.createClass( {
 			onSaveDraft: noop,
 			post: null,
 			savedPost: null,
-			site: {}
+			site: {},
+			user: null,
+			userUtils: null,
 		};
 	},
 
+	componentDidMount: function() {
+		if ( ! config( 'email_verification_gate' ) ) return;
+
+		this.props.user
```

Comment:
You should include your `this.props.user` check in the early return line above, here and throughout the file.